src/netaddress.h

```diff
@@ -49,18 +51,50 @@ enum Network
     NET_MAX,
 };
 
+/// If an IPv6 address begins with this, then we treat the rest of it as IPv4 address.
+static constexpr uint8_t IPv4_IN_IPv6_PREFIX[12] = {
```

Comment:
Nit: uppercase v in global constants (here and further).

Reply:
Changed to uppercase.